automated/linux/alsa-smoke/alsa-smoke.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+
+# shellcheck disable=SC1091
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+RESULT_FILE="${OUTPUT}/result.txt"
+export RESULT_FILE
+REFERENCE_PATH="/dev/snd"
+
+usage() {
+    echo "Usage: $0 [-s <true|false>] [-p </path/to/snd/devices>]" 1>&2
+    exit 1
+}
+
+while getopts "s:p:d:h" o; do
```

Comment:
'd' isn't used.

Reply:
Fixed. I used it for something in one of the previous versions, but now I forgot what it was.